content/lua/concepts/mathematical-library/terms/asin/asin.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Lua .asin() Method
```

Comment:
the metadata section is incorrect. Please reference a template or one of our existing entries to see the correct formatting and elements
https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md?plain=1